internal/pkg/api/handleCheckin.go

```diff
@@ -454,22 +455,21 @@ func (ct *CheckinT) processUpgradeDetails(ctx context.Context, agent *model.Agen
 		return nil
 	}
 	// update docs with in progress details
-
 	vSpan, vCtx := apm.StartSpan(ctx, "Check update action", "validate")
-	action, err := ct.verifyActionExists(ctx, vSpan, agent, details)
-	if err != nil {
+
+	var actionTraceparent string
```

Comment:
@blakerouse normally when no actin is present we stop here
but we make loading optional and if action is not present we just set traceparent (only property used here in the code) to ""

then later we check if we have here when setting parent span and continue until we reach a point where we update details on line 543